src/Browser.ts

```diff
@@ -418,7 +412,7 @@ export class Browser {
         this.#fkey = fkey;
 
         if (typeof fkey === "undefined") {
-            throw new InternalError("Unable to find fkey.");
+            throw new InternalError(`Unable to find fkey, got: ${$.html()}`);
```

Comment:
IDK if it's a good habit of dumping HTML as the error.

If people use try/catch, it's much harder to detect errors based on dynamic text. Maybe pass the HTML as a debug parameter to `InternalError`?

Reply:
Good question - the reason I added it is actually because when I was debugging a problem with `fkey`, the message "unable to find fkey" wasn't too helpful as to *why* it cannot be found. However, I feel a bit iffy about dumping the HTML string too, and like the idea of exposing it as a property on `InternalError` instead.